<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JobXSISettingsWidgetUI</class>
 <widget class="QGroupBox" name="JobXSISettingsWidgetUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="title">
   <string>XSI Settings</string>
  </property>
  <layout class="QVBoxLayout" name="mVerticalLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <item>
    <layout class="QGridLayout" name="_2">
     <property name="margin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>-1</number>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="textLabel9">
       <property name="text">
        <string>Width</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="FieldSpinBox" name="mHeightSpin">
       <property name="maximum">
        <number>1000000</number>
       </property>
       <property name="field" stdset="0">
        <string>height</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="textLabel8">
       <property name="text">
        <string>Height</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="FieldLineEdit" name="mPassEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="field" stdset="0">
        <string>pass</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Pass:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="mRendererCombo">
       <item>
        <property name="text">
         <string>Mental Ray</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Scanline - Software</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Scanline - OpenGL</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Scanline - OpenGL - Rapid(Large Scenes)</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Renderer:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="FieldSpinBox" name="mWidthSpin">
       <property name="maximum">
        <number>1000000</number>
       </property>
       <property name="field" stdset="0">
        <string>width</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox3">
     <property name="title">
      <string>Flags</string>
     </property>
     <layout class="QVBoxLayout">
      <property name="spacing">
       <number>-1</number>
      </property>
      <property name="margin">
       <number>9</number>
      </property>
      <item>
       <widget class="FieldCheckBox" name="mMotionBlurCheck">
        <property name="text">
         <string>Motion Blur</string>
        </property>
        <property name="field" stdset="0">
         <string>motionBlur</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="FieldCheckBox" name="mDeformMotionBlurCheck">
        <property name="text">
         <string>Deform Motion Blur</string>
        </property>
        <property name="field" stdset="0">
         <string>deformMotionBlur</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>337</width>
       <height>62</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FieldLineEdit</class>
   <extends>QLineEdit</extends>
   <header>fieldlineedit.h</header>
  </customwidget>
  <customwidget>
   <class>FieldSpinBox</class>
   <extends>QSpinBox</extends>
   <header>fieldspinbox.h</header>
  </customwidget>
  <customwidget>
   <class>FieldCheckBox</class>
   <extends>QCheckBox</extends>
   <header>fieldcheckbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
